src/main/java/org/sonar/plugins/stash/issue/StashComment.java

```diff
@@ -53,6 +63,27 @@ public long getVersion() {
     return version;
   }
   
+  public List<StashTask> getTasks() {
+    return tasks;
+  }
+  
+  public void addTask(StashTask task){
+    tasks.add(task);
+  }
+  
+  public boolean containsNotDeletableTasks() {

```

Comment:
Suggestion: avoid negation
public boolean containsDeletableTasks() {
    boolean result = true;

    for (StashTask task: tasks){
      if (! task.isDeletable()){
        result = false;
        break;
      }
    }

    return result;
  }